pkg/imager/profile/profile.go

```diff
@@ -67,7 +79,7 @@ func (p *Profile) Validate() error {
 	}
 
 	if p.Board != "" {
-		if !(p.Arch == arm64 && p.Platform == "metal") {
+		if p.Arch != arm64 || p.Platform != "metal" {
```

Comment:
we could also validate here that `Board` should not be set if we support overlays, or via TODO